### DISCUSSION TITLE
umijs 是否支持webpack5的resolve.fallback属性，当正常解析失败时，重定向模块请求。

### DISCUSSION THREAD

**Discussion by ljmCoder** [umijs/umi#11750] [Q&A]
umijs 是否支持webpack5的resolve.fallback属性，“当正常解析失败时，重定向模块请求”

尝试通过chainwebpack进行配置。 发现chainwebpack目前并不支持这一操作。

请教各位，是我使用错误？或者有其他解决办法吗

![image](https://github.com/umijs/umi/assets/41459781/a971b2c7-ca5f-452e-98e3-2ec85ae6617e)

**Comment by fz6m** [comment]
你要 fallback 啥，直接自定义 alias 到你最终位置就好了。

如有上下文环境，需要给一个最小复现。
